events/wc_druidic_events.txt

```diff
@@ -38,14 +59,18 @@ character_event = {
 			factor = 2
 			any_war = { war_score = -25 }
 		}
+		modifier = {
+			factor = 10
+			any_neighbor_independent_ruler = { religion = death_god }
```

Comment:
I wouldn't recommend to use `any` scopes in `weight_multiplier`. It's CPU heavy.